views/pages/contactUs.php

```diff
@@ -0,0 +1,42 @@
+<?php session_start(); ?>
```

Comment:
We can look at making sure this page is in the MVC model on Tuesday, when we all meet up,